examples/base_and_running_difference_mapcubes.py

```diff
@@ -0,0 +1,60 @@
+"""
+=======================================
+Base and Running Difference in Mapcubes
+=======================================
+
+This example illustrates how to do base and running differencing with a MapCube.
+Base differencing uses a fixed reference point when compared to running difference.
+"""
+
+import matplotlib.pyplot as plt
+import matplotlib.colors as colors
+import matplotlib.cm as cm
+
+import sunpy.map
+import sunpy.physics.differential_rotation as diffrot
+from sunpy.data.sample import AIA_193_CUTOUT01_IMAGE, AIA_193_CUTOUT02_IMAGE, AIA_193_CUTOUT03_IMAGE
+
+###############################################################################
+# We create the MapCube using the AIA_193_CUTOUT sample data.
+# To create a MapCube, we can call Map directly but add in a keyword to output a MapCube instead.
```

Comment:
To create a MapCube, we can call Map directly and add in a keyword to output a MapCube instead.